webplugin/css/app/mck-sidebox-1.0.css

```diff
@@ -5358,24 +5358,24 @@ div#mck-rated:before{
 	right: 80%;
 }
 .mck-feedback-text-wrapper{
-	max-height: 80px;
-    background: white;
     overflow: hidden;
     display: flex;
     align-items: center;
     justify-content: flex-start;
     border-radius: 3px;
     margin-top: 18px;
     padding: 7px;
-    border: solid 0.5px #cecbcb;
     max-width: 90%;
-    margin: 18px auto;
+	margin: 18px auto;
```

Comment:
code formatting

Reply:
its some bug with GitHub formatting looks fine on vscode

Reply:
see if you have tab as 8 space vs 4 space setting?

Reply:
Its set at 4space only